lab_4_retrieval_w_clustering/main.py

```diff
@@ -100,6 +130,11 @@ def __init__(self, stop_words: list[str]) -> None:
         Args:
             stop_words (list[str]): List with stop words.
         """
+        self.stop_words = stop_words
```

Comment:
useless, just put them in Tokenizer initially

Reply:
why? how can i do it if it is needed as the object in init

Reply:
no, just put stop_words in Tokenizer initially.

Reply:
oh, fixed!!